incuna-sass/_normalize.sass

```diff
@@ -1,5 +1,5 @@
 // Incuna overrides
-@if not support-legacy-browser('ie', '7')
+.lt-ie8 &
```

Comment:
all these files still need to be documented so I agree that these should be in there. Possibly making a readme for each folder could work